### PR TITLE
DMEERX-1042

### PR DESCRIPTION
**Pull request by yuliumitre**
This PR addresses several subtasks for https://jira.mitre.org/secure/RapidBoard.jspa?projectKey=DMEERX&rapidView=772:

https://jira.mitre.org/browse/DMEERX-1051
https://jira.mitre.org/browse/DMEERX-1067
https://jira.mitre.org/browse/DMEERX-1069

To test the PR, would need to use different branches other than master:
CRD: cql-and-question-pages
CDS-Library: adform-cql-multiple-questions
DTR: cql

Steps to test:
1. Select patient with Home Oxygen Therapy order
2. There will be a form for "Additional Form"
3. Click that until "Next questions" button shows up
4. Click "Next questions", then picking different answer based on the question
5. Click "Send to payer" if the form is complete

Another set of test is related to save the adaptive form to EHR:
1. During completing the adaptive form (in-progress), the user can click "Save To EHR"
2. Next time when the user click the "Additional Form", they have the option to load those "in-progress" form
3. Once the saved form is loaded, the user can continue to get next question as before.

### CODE REVIEW

**Comment by KeeyanGhoreshi** [comment]
I'm getting an error when clicking "Save to EHR".  It seems like it still saves to the EHR, but it fails in showing a popup to say it was successful I'm assuming.

There's also some inconsistent behavior with the "Submit the order form" instruction, it doesn't show up if you choose "Change in Status".  

Also the test question, "Another additional test question that should be displayed with Question 1.4", does not get filtered even if it is answered.  Not sure if this intended.

**Comment by yuliumitre** [comment]
> I'm getting an error when clicking "Save to EHR". It seems like it still saves to the EHR, but it fails in showing a popup to say it was successful I'm assuming.
> @KeeyanGhoreshi what error did you get? Can you provide some details? Thanks. 
> There's also some inconsistent behavior with the "Submit the order form" instruction, it doesn't show up if you choose "Change in Status".
> @RobertScalfani Do you know what is going on here? 
> Also the test question, "Another additional test question that should be displayed with Question 1.4", does not get filtered even if it is answered. Not sure if this is intended.

**Comment by KeeyanGhoreshi** [comment]
The error I'm getting is `Uncaught TypeError: this.clickChild is not a function`

**Comment by yuliumitre** [comment]
> The error I'm getting is `Uncaught TypeError: this.clickChild is not a function`

Got it. Thanks and I will take a look. I saw this behavior inconsistently.

**Review by KeeyanGhoreshi — COMMENTED** [review]
Works well, not sure why the docker build is failing.

File: src/App.jsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React, { Component } from "react";
+import React, { Component, useCallback } from "react";
```
[the remark this answers is not in the document]
Comment (yuliumitre) [reply src/App.jsx:1]: Hmm, it is not. Will remove it. Thanks.

[the remark this answers is not in the document]
Comment (yuliumitre) [reply src/App.jsx]: The latest docker build passed. It was probably a fluke :-) ? @KeeyanGhoreshi

**Review by KeeyanGhoreshi — APPROVED** [review]

**Pull request merged**